styx-common/src/test/java/com/spotify/styx/storage/DatastoreStorageTest.java

```diff
@@ -452,15 +372,15 @@ public void allFieldsAreSetWhenRetrievingWorkflowState() throws Exception {
         WorkflowConfiguration.builder()
             .id(WORKFLOW_ID1.id())
             .schedule(DAYS)
+            .dockerImage(DOCKER_IMAGE.get())
+            .commitSha(COMMIT_SHA)
             .build()));
     Instant instant = Instant.parse("2016-03-14T14:00:00Z");
     Instant offset = instant.plus(1, ChronoUnit.DAYS);
     TriggerInstantSpec spec = TriggerInstantSpec.create(instant, offset);
     storage.updateNextNaturalTrigger(WORKFLOW_ID1, spec);
     WorkflowState state = WorkflowState.builder()
         .enabled(true)
-        .dockerImage(DOCKER_IMAGE.get())
-        .commitSha(COMMIT_SHA)
```

Comment:
We should also check that the `Workflow` returned from Storage is the same as the one constructed above (so we also verify that dockerImage and commitSha have been set properly by `store` operation).

Reply:
not sure about that, there are other tests that already do that